apps/wallet/src/dapp-interface/WalletStandardInterface.ts

```diff
@@ -244,6 +252,15 @@ export class SuiWallet implements Wallet {
 		return { accounts: this.accounts };
 	};
 
+	#disconnect: StandardDisconnectMethod = async () => {
+		this.#accounts = [];
+		this.#events.emit('change', { accounts: this.accounts });
```

Comment:
I think wallet will notify the dapp when it disconnects the accounts. Is this necessary?

Reply:
Fair point. It should all be picked up by the `WalletStatusChangePayload` listener!